app/Policies/ServiceTemplatePolicy.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Policies;
+
+use App\Facades\Permissions;
+use App\Models\ServiceTemplate;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class ServiceTemplatePolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any service templates.
+     *
+     * @param \App\Models\User $user
+     * @return mixed
+     */
+    public function viewAny(User $user)
+    {
+        return $user->hasGlobalRead();
+    }
+
+    /**
+     * Determine whether the user can view the service template.
+     *
+     * @param \App\Models\User $user
+     * @param \App\Models\ServiceTemplate $template
+     * @return mixed
+     */
+    public function view(User $user, ServiceTemplate $template)
+    {
+        return $this->viewAny($user) || Permissions::canAccessServiceTemplate($template, $user);
```

Comment:
I doubt i did anything intentionally, (sorry it's been 2+ months since i looked at this code properly). I imagine i might have used something else as an example or template, feel free to suggest a change if it doesn't seem right?

Reply:
My suggestion is to remove per-user permissions for now as they don't add much.  At some point in the future, it will be possible to add fine grained user permissions.  If this exists as is the migration will be more complicated.